.gitmodules

```diff
@@ -9,4 +9,4 @@
 [submodule "era-contracts"]
 	path = era-contracts
 	url = https://github.com/lambdaclass/era-contracts
```

Comment:
Should we also need to update url, since branch is in https://github.com/matter-labs/era-contracts repo?

Reply:
Yes, please make sure to run `git submodule set-url era-contracts https://github.com/matter-labs/era-contracts`.
Otherwise I have had issues with Git unable to pick up the origin change.